src/lib/formatters/html-report/report.hbs

```diff
@@ -0,0 +1,105 @@
+        <title>Sonar Report</title>
+        <style type="text/css">
+            body {
+                font: 14px Consolas, Courier, monospace;
+            }
+
+            h1,h2,h3,h4,p {
+                margin: 0;
+                padding: 0;
+            }
+
+            .highlight {
+                color: #111111;
+                font-weight: bold;
+            }
+
+            .container.error {
+                background-color: #ff9999;
+            }
+
+            .container.warning {
+                background-color: #ffff66;
+            }
+
+            .step {
+                margin: 10px;
+            }
+
+            .step .text {
+                border-radius: 3px;
+                color: #666666;
+                padding: 5px;
+                display: block;
+            }
+
+            .container {
+                background: #f8f8f8;
+                border-radius: 5px;
+                border: 1px solid #e8e8e8;
+                padding: 10px;
+                margin: 10px;
+            }
+        </style>
+        <meta charset="UTF-8">
+    </head>
+
+    <body>
+        {{#each resources}}
+        <div class="feature container">
+            <h3 class="title"><span class="highlight">Resource: </span>{{resource}}</h3>
+            <p>Total Errors: {{totalErrors}}; Total Warnings: {{totalWarnings}}</p>
+            {{#each ruleIds}}
+            <div class="element container">
+                <h3 class="title"><span class="highlight">Rule: </span>
+                    <a href="https://sonarwhal.com/docs/user-guide/rules/{{ruleId}}.html">
+                        {{ruleId}}
+                    </a>
+                </h3>
+                {{#each issues}}
+                <div class="container {{getSeverityClass severity}}">
+                    {{#if location}}
+                    <div class="step">
+                        <p>
+                            <span class="text">
+                                <span class="keyword highlight">Location</span>: line:{{location.line}};
+                            col:{{location.column}}
+                            </span>
+                        </p>
+                    </div>
+                    {{/if}}
+                    <div class="step">
+                        <p>
+                            <span class="text">
+                                <span class="keyword highlight">message:</span>
+                            </span>
+                        </p>
+                        <div id="error">
+                            {{message}}
+                        </div>
+                    </div>
+                    {{#if sourceCode}}
+                    <div class="step">
+                        <p>
+                            <span class="text">
+                                <span class="keyword highlight">Source Code:</span>
+                            </span>
+                        </p>
+                        <div id="error">
```

Comment:
You already have `id="error"` at [line 81](https://github.com/sonarwhal/sonar/pull/411/files#diff-55b1f3586372d9f9c51a10c789eba303R81).